src/patternfly/components/CalendarMonth/calendar-month.scss

```diff
@@ -0,0 +1,226 @@
+.pf-c-calendar-month {
+  --pf-c-calendar-month--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-calendar-month--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-calendar-month--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-calendar-month--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-calendar-month--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-calendar-month--FontSize: var(--pf-global--FontSize--sm);
+
+  // header
+  --pf-c-calendar-month__header--MarginBottom: var(--pf-global--spacer--md);
+  --pf-c-calendar-month__header-year--Width: 8ch;
+  --pf-c-calendar-month__header-nav--MarginRight: 0;
+  --pf-c-calendar-month__header-nav--MarginLeft: 0;
+  --pf-c-calendar-month__header-nav--m-prev-month--MarginRight: var(--pf-global--spacer--sm);
```

Comment:
why does this need a bigger margin when the nav is for the prev-month?

Reply:
Hmmm... not sure what you mean. The prev/next button shave the same margin from the month/year elements, and both have negative margins to align the arrow with the content in the box.

<img width="363" alt="Screen Shot 2020-11-04 at 9 35 44 AM" src="https://user-images.githubusercontent.com/35148959/98131811-45776880-1e81-11eb-93f0-a2684c13cc0b.png">
<img width="390" alt="Screen Shot 2020-11-04 at 9 35 28 AM" src="https://user-images.githubusercontent.com/35148959/98131813-460fff00-1e81-11eb-8f83-ebcc7eb17e84.png">

Reply:
Not sure if this is relevant or not, but I'm putting this component inside a Popover which seems to already have some content padding: https://patternfly-react-pr-5026.surge.sh/components/date-picker

Reply:
@redallen would we not use the date picker for that? https://www.patternfly.org/v4/components/date-picker

Reply:
That's the date picker I'm refactoring to not use `flatpickr` :)

Reply:
Oh I mean the date picker was built to hold any sort of calendar, not just flatpickr. Could we not continue to use the date picker component with the calendar month component inside of it?

Reply:
Unfortunately the current date picker in React is built to hold _exactly_ `flatpickr` which creates its own version of a popover. I've transitioned to just using the `Popover` component in https://github.com/patternfly/patternfly-react/pull/5026 , but it comes with 16px of padding. I can find a way to remove that padding in favor of the `24 24 16 24` in this PR if needbe, just thought I'd let you know!